docs/source/index.rst

```diff
@@ -1,53 +1,54 @@
 
 Augur is focused on making sense of data using
-four key **human centered data science** strategies: 
+four key **human centered data science** strategies:
 
-1. **Enable comparisons**. People navigate complex unknowns analogically. Let folks see how their project compares with others they are familiar with. This is not ranking - if you start thinking about "metrics" like "rankings," you are probably going to create suboptimal metrics. 
-2. Make **time a fundamental dimension in all metrics** from the start. "Point in time scores" are useful. They are more useful if we can see how they compare historically and can be used to anticipate a trajectory. 
-3. All **data** driving visualizations **should be downloadable** as ``.csv`` or some other data exchange format. People trust metrics when they can see the underlying data, and proving traceability back to the CHAOSS Project's metrics standards requires easy transparency. 
+1. **Enable comparisons**. People navigate complex unknowns analogically. Let folks see how their project compares with others they are familiar with. This is not ranking - if you start thinking about "metrics" like "rankings," you are probably going to create suboptimal metrics.
+2. Make **time a fundamental dimension in all metrics** from the start. "Point in time scores" are useful. They are more useful if we can see how they compare historically and can be used to anticipate a trajectory.
+3. All **data** driving visualizations **should be downloadable** as ``.csv`` or some other data exchange format. People trust metrics when they can see the underlying data, and proving traceability back to the CHAOSS Project's metrics standards requires easy transparency.
 4. Make **all the visualizations downloadable as .svg's**. People want to put your visualizations in reports to explain things they care about. And ask for money. Give them the tools. That's what makes folks care about metrics.
 
 Our core team has a long standing interest in social computing, software engineering measurement and the ethical instrumentation of online human behavior.
 
-Our members:
+Current members:
 
 - `Derek Howard <https://howderek.com>`_
 - `Sean P. Goggins <http://www.seangoggins.net>`_
 - `Matt Germonprez <https://goo.gl/E87KdK>`_
 - `Gabe Heim <https://github.com/gabe-heim>`_
 - `Matt Snell <https://github.com/Nebrethar>`_
+- `Max Balk <https://github.com/maxbalk/>`_
+- `Andrew Brain <https://github.com/ABrain7710>`_
+- Ryland Dohm
+- `Carter Landis <https://github.com/ccarterlandis>`_
+
+Former members:
+
 - `Jonah Zukosky <https://github.com/jonahz5222>`_
 - `Carolyn Perniciaro <https://github.com/CMPerniciaro>`_
 - `Elita Nelson <https://github.com/ElitaNelson>`_
 - `Michael Woodruff <https://github.com/michaelwoodruffdev/>`_
```

Comment:
Should probably acknowledge Trey. And we should probably call them "Current Core Contributors" and "Previous Core Contributors". .. And we should probably include our GSOC students. I would say we call everyone except Keanue a "Current Core Contributor", as even Parth is still doing a few things here and there.

Reply:
Also, @ChristianCme should be added to former core contributors, IMHO.

Reply:
Oh, nevermind on @howderek ... he's at the top ... and I think @sgoggins and @germonprez should probably be at the bottom of the list if its not alphabetical. Other current core contributors are, I think @geekygirldawn and @ivanayov ....

Reply:
There is no set order